src/altinn-app-frontend/src/features/form/layout/update/updateFormLayoutSagas.ts

```diff
@@ -589,7 +589,9 @@ export function* updateRepeatingGroupEditIndexSaga({
       const currentView = state.formLayout.uiConfig.currentView;
       const onlyInRowIndex =
         validate === Triggers.ValidateRow
-          ? state.formLayout.uiConfig.repeatingGroups[group].editIndex
+          ? (state.formLayout.uiConfig.repeatingGroups &&
+              state.formLayout.uiConfig.repeatingGroups[group].editIndex) ||
+            undefined
```

Comment:
if `editIndex=0` this will now fail, as `0 || undefined = undefined`

Reply:
You're absolutely right. I shudder to think how many of those mistakes I made in #597.. But functionality without automatic tests is never critical, right? :wink:

Reply:
Hehe right, lucky the test used editIndex=0 😅